.eleventy.js

```diff
@@ -28,16 +28,16 @@ module.exports = function(eleventyConfig) {
   };
   eleventyConfig.addPlugin(cacheBuster(cacheBusterOptions));
 
-  // Add CSS minifier
-  eleventyConfig.addFilter("cssmin", function(code) {
-    return new CleanCSS({}).minify(code).styles;
-  });
+  // Add CSS minifier -- replaced by a gulp task
+  // eleventyConfig.addFilter("cssmin", function(code) {
+  //   return new CleanCSS({}).minify(code).styles;
+  // });
 
   // Set directories to pass through to the dist folder
-  eleventyConfig.addPassthroughCopy('./src/img/');
-  eleventyConfig.addPassthroughCopy('./src/fonts/');
-  eleventyConfig.addPassthroughCopy('./src/js/');
-  eleventyConfig.addPassthroughCopy('./src/favicons/');
+  eleventyConfig.addPassthroughCopy('./src/assets/img/');
+  eleventyConfig.addPassthroughCopy('./src/assets/fonts/');
+  eleventyConfig.addPassthroughCopy('./src/assets/js/');
+  eleventyConfig.addPassthroughCopy('./src/assets/favicons/');
```

Comment:
Agreed that we should move everything to an `assets/` directory; could we do this with the `main.css` file as well? You will also need to update the path in `<head>`